lib/modules/manager/maven-wrapper/extract.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { loadFixture } from '../../../../test/util';
+import { extractPackageFile } from './extract';
```

Comment:
```suggestion
import { extractPackageFile } from '.';
```

better code coverage